modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -51,6 +52,12 @@ protected void configure() {
 
         withEnv("SERVICES", services.stream().map(Service::getLocalStackName).collect(Collectors.joining(",")));
 
+        if (hostnameExternal != null) {
+            withEnv("HOSTNAME_EXTERNAL", hostnameExternal);
+        } else {
+            withEnv("HOSTNAME_EXTERNAL", getContainerIpAddress());
```

Comment:
I think this is relevant for my comment on #1413. 

For S3, to support hostname based bucket paths, we use `IPADDRESS.nip.io`, so that `BUCKETNAME.IPADDRESS.nip.io` resolves to just `IPADDRESS`.

With the latest localstack I suspect `HOSTNAME_EXTERNAL` needs to be set to `IPADDRESS.nip.io` by default.

If this is true, I wonder if we should change this line to `withEnv("HOSTNAME_EXTERNAL", getContainerIpAddress() + "nip.io")`. Or, so that other services work happily, we might need to override `getContainerIpAddress()` to return `super.getContainerIpAddress() + "nip.io"` (or similar)...

Reply:
Ignore this comment; I think #1504 makes it totally redundant.